guides/source/getting_started.md

```diff
@@ -334,216 +340,650 @@ If we go back to our browser and make a request to
+is not saved to the database at all, it's just available in the console so far.
+To save the object to the database, we need to call `save`:
+
+```ruby
+irb(main):002:0> article.save
+```
+
+This command will show us the following output:
+
+```plaintext
+(0.1ms)  begin transaction
+Article Create (0.4ms)  INSERT INTO "articles" ("title", "body", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["title", "Hello Rails"], ["body", "I am on Rails!"], ["created_at", "2020-01-18 23:47:30.734416"], ["updated_at", "2020-01-18 23:47:30.734416"]]
+(0.9ms)  commit transaction
+=> true
+```
+
+This output shows an `INSERT INTO "articles"...` database query. This means
+that our article has been successfully inserted into our table.
+
+If we take a look at our `article` object again, an interesting thing has happened:
+
+```ruby
+irb(main):003:0> article
+=> #<Article id: 1, title: "Hello Rails", body: "I am on Rails!", created_at: "2020-01-18 23:47:30", updated_at: "2020-01-18 23:47:30">
 ```
 
-If you open up the newly generated `app/controllers/articles_controller.rb`
-you'll see a fairly empty controller:
+Our object now has the `id`, `created_at` and `updated_at` fields set. All of
+this happened automatically for us when we saved this article.
+
+If we wanted to retrieve this article back from the database later on, we can
+do that with `find`, and pass that `id` as an argument:
+
+```ruby
+irb(main):004:0> article = Article.find(1)
+=> #<Article id: 1, title: "Hello Rails", body: "I am on Rails!", created_at: "2020-01-18 23:47:30", updated_at: "2020-01-18 23:47:30">
+```
+
+A shorter way to add articles into our database is to use `Article.create`, like this:
+
+```ruby
+irb(main):005:0> Article.create(title: "Post #2", body: "Still riding the Rails!")
+```
+
+This way, we don't need to call `new` and then `save`.
+
+Lastly, models provide a method to find all of their data:
+
+```ruby
+irb(main):006:0> articles = Article.all
+ #<ActiveRecord::Relation [
+   #<Article id: 1, title: "Hello Rails", body: "I am on Rails!", created_at: "2020-01-18 23:47:30", updated_at: "2020-01-18 23:47:30">,
+   #<Article id: 2, title: "Post #2", body: "Still riding the Rails!", created_at: "2020-01-18 23:53:45", updated_at: "2020-01-18 23:53:45">]>
+```
+
+This method returns an `ActiveRecord::Relation` object, which you can think of
+as a super-powered array. This array contains both of the topics that we have
+created so far.
+
+TIP: For more about models, read the [Active Record
+Basics](https://guides.rubyonrails.org/active_record_basics.html) and [Active
+Record Query
+Interface](https://guides.rubyonrails.org/active_record_querying.html) guides.
+
+As you can see, models are very helpful classes for interacting with databases
+within Rails applications. Models are the final piece of the "MVC" puzzle.
+Let's look at how we can go about connecting all these pieces together into a
+cohesive whole.
```

Comment:
I think this last sentence would read better as "Let's look at how we can go about connecting all these pieces together." - especially for those where english is their second language, cohesive might not be a word they know. I think the sentence says what we need to say without "into a cohesive whole"